dev-setup/ubuntu/ubuntu-2004/SetupVMTest.txt

```diff
@@ -0,0 +1,105 @@
+#VM 20.04 Setup
+
+##Pre-Install
+
+        sudo apt-get update && sudo apt-get install -y apt-transport-https ca-certificates
+        sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys CE7709D068DB5E88 || sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys CE7709D068DB5E88
+        sudo echo "deb https://repo.sovrin.org/deb bionic master" >> /etc/apt/sources.list
+        sudo echo "deb https://repo.sovrin.org/deb bionic stable" >> /etc/apt/sources.list
+        sudo echo "deb http://security.ubuntu.com/ubuntu bionic-security main" >> /etc/apt/sources.list
+
+        sudo apt-get update && sudo apt-get install -y \
+            git \
+            wget \ 
+            unzip \ 
+            python3-pip \ 
+            python3-venv \ 
+            libsodium23 \ 
+            iptables \ 
+            at \ 
+            supervisor \ 
+            python3-nacl \ 
+            rocksdb-tools \ 
+            librocksdb5.17 \ 
+            librocksdb-dev \ 
+            libsnappy-dev \ 
+            liblz4-dev \ 
+            libbz2-dev \ 
+            libssl1.0.0 \ 
+            libindy \ 
+            ursa
+
+        git clone https://github.com/hyperledger/indy-node.git
+        git clone https://github.com/hyperledger/indy-plenum.git
+        # in both indy-node and indy-plenum checkout origin/ubuntu-20.04-upgrade 
+        sudo cp /usr/lib/ursa/libursa.* /usr/lib/
+        # Should be done in python env
+        pip install -U \
+            Pygments==2.2.0 \
```

Comment:
This needs to be upgraded.

Reply:
@WadeBarnes I totally agree with you, we need to update Pygments version. But Indy Node does not have a pinned dependency on Pygments package. And this file is just a set of recommendations for a virtual machine, relevant at the moment.
Thus, I propose not to block this PR and merge it, since Indy Node has nothing to do with Pygments package. Only after updating this package in Plenum does it make sense to update or delete this doc.